msteams-platform/graph-api/proactive-bots-&-messages/graph-proactive-bots-&-messages.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Use Microsoft Graph to enable proactive bot installation and messaging in Teams
```

Comment:
How do you find this document? Is there a table of contents somewhere? How does this doc relate to https://docs.microsoft.com/en-us/microsoftteams/platform/bots/how-to/conversations/send-proactive-messages?tabs=dotnet ?

Reply:
Updated the TOC; the above doc is a guide for v3 Bot Framework SDK.